### DISCUSSION TITLE
Stats output

### DISCUSSION THREAD

**Discussion by ksuchacki** [ENHANCE-PET/MOOSE#88] [Q&A]
The stats output has the following headings: 
Mean, StDev, Median, Max and Min. 

What the the units of the mean, median max and min? Is there a way to export the SUV results from the PET data after using MOOSE to segment the CT images? Many thanks, Karla.

**Comment by LalithShiyam** [comment]
@ksuchacki - I am not sure how I missed this. I am extremely sorry! If you are using the `moosev2` it's whatever unit your PET images are. If your PET image is already an SUV image, you will have these values in SUV, if it's in counts or kBq/mL, you need to convert them on your own.

I am terribly sorry - still not sure how I missed this!
